### DISCUSSION TITLE
Import package

### DISCUSSION THREAD

**Discussion by anil88** [pion/webrtc#2387] [Q&A] [answered]
I am new in go, I need help for basic use case. How can I import the package?
I tried: 
go install github.com/pion/webrtc/v3/pkg/media@latest
go install github.com/pion/webrtc/v3/pkg@latest
go install github.com/pion/webrtc/v3@latest
go install github.com/pion/webrtc@latest

I get always "package github.com/pion/webrtc is not a main package" or 
"go: github.com/pions/webrtc/v3@latest: github.com/pion/webrtc/v3@v3.1.50: parsing go.mod:
        module declares its path as: github.com/pions/webrtc/v3
                but was required as: github.com/pion/webrtc/v3"
And if I try with pion**s** it still not work.

**Comment by Sean-Der** [comment, marked as answer]
Hi @anil88 what version of golang are you using? My guess is that you are using a version of go before modules were supported. Pion requires modules unfortunately. 

If you run/build anything in the `examples` directory does it work?
